python/MOCK_AIS.py

```diff
@@ -12,7 +12,7 @@
 
 # Constants
 AIS_PUBLISH_PERIOD_SECONDS = 0.1  # Keep below 1.0 for smoother boat motion
-NUM_AIS_SHIPS = 30
+NUM_AIS_SHIPS = rospy.get_param('num_ais_ships', '5')
```

Comment:
I think the default value should be `5` instead of `'5'` so that it will be an int, not a string